tox.ini

```diff
@@ -45,7 +48,7 @@ deps =
     -r{toxinidir}/requirements/postgres.txt
 commands =
     py.test {posargs:--cov=kolibri --color=no}
-    rm -r {env:KOLIBRI_HOME}
+    rm -rf {env:KOLIBRI_HOME}
```

Comment:
heh, that was a left-over... I thought the `-f` for force would help out, but I haven't actually seen deletion problems in the postgres tests... I guess it's because it doesn't run `kolibri stop`